dev/parm/config/sfs/config.base.j2

```diff
@@ -318,23 +318,8 @@ export DO_METP="NO"         # Run METPLUS jobs - set METPLUS settings in config.
 export DO_FIT2OBS="NO"      # Run fit to observations package
 
 # Archiving options
-export HPSSARCH="{{ HPSSARCH }}"        # save data to HPSS archive
-export LOCALARCH="{{ LOCALARCH }}"        # save data to local archive
-if [[ ${HPSSARCH} = "YES" ]] && [[ ${LOCALARCH} = "YES" ]]; then
-   echo "Both HPSS and local archiving selected.  Please choose one or the other."
-   exit 3
-elif [[ ${HPSSARCH} = "YES" ]] || [[ ${LOCALARCH} = "YES" ]]; then
-   export DO_ARCHCOM="YES"
-else
-   export DO_ARCHCOM="NO"
-fi
-export ARCH_CYC=00           # Archive data at this cycle for warm start and/or forecast-only capabilities
-export ARCH_WARMICFREQ=4     # Archive frequency in days for warm start capability
-export ARCH_FCSTICFREQ=1     # Archive frequency in days for gdas and gfs forecast-only capability
-export ARCH_EXPDIR='YES'     # Archive the EXPDIR configs, XML, and database
-export ARCH_EXPDIR_FREQ=0    # How often to archive the EXPDIR in hours or 0 for first and last cycle only
-export ARCH_HASHES='YES'     # Archive the hashes of the GW and submodules and 'git status' for each; requires ARCH_EXPDIR
-export ARCH_DIFFS='NO'       # Archive the output of 'git diff' for the GW; requires ARCH_EXPDIR
+export DO_ARCHCOM="{{ DO_ARCHCOM }}"    # Tar and archive the COM directories
+export ARCHCOM_TO="{{ ARCHCOM_TO }}"    # Valid options are hpss, globus_hpss, and local
```

Comment:
@NeilBarton-NOAA These lines are triggering the `arch_tars` dependency on the `sfs_cleanup` task.  Since `sfs_arch_tars` does not exist, this is causing a stall.  Can you please set `DO_ARCHCOM` to NO for now and reinstate these lines on the next PR that enables archiving?